templates/takeovers/_openstack_security_takeover.html

```diff
@@ -0,0 +1,79 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row">
+    <div class="p-takeover__body u-clearfix u-vertically-center">
+      <div class="col-8">
+        <h1 class="p-takeover__title">Private cloud security</h1>
+        <p class="p-heading--four u-no-margin--right">
+          Meeting the highest enterprise requirements with OpenStack.
+        </p>
+        <ul class="p-inline-list u-hide--small">
+          <li class="p-inline-list__item">
+            <!-- Update link -->
+            <a
+              href="/engage/openstack-security-webinar?utm_source=Takeover &utm_medium=Takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```

Comment:
Please update the link to the landing page we want so it 404's and the linkchecker blocks it merging.